### ISSUE TITLE
Extra directories include appdata path

### ISSUE THREAD

**Issue by nethminagit** [nickrod518/Migrate-WindowsUserProfile#63]
Thank you for this excellent GUI

This not a issue but a question...

Is there a way to specify appdata into the extra directories section. I only wish to extract certain paths and not the entire appdata directory?

**Comment by nickrod518** [comment]
You should be able to uncheck the AppData option under "Data to include" and then hit the "+" sign under "Extra directories to include" for each subdirectory of appdata that you desire.

**Comment by mgibbonstech** [comment]
Would it be possible to have the ability to include only certain directories in the local app data?  I don't want to include the entire local app data folder, just the Google Chrome folder.

**Comment by nickrod518** [comment]
Yes, just add the folder in the additional directories option instead of ticking the localappdata option.

Thanks,
Nick

**Comment by mgibbonstech** [comment]
Maybe I'm missing something, but I don't see the local app data folder when i click on additional directories.

**Comment by nickrod518** [comment]
You should be able to manually add the path. Maybe you need to enable hidden files?

Thanks,
Nick
